client/components/fixed-navigation-header/docs/example.jsx

```diff
@@ -0,0 +1,15 @@
+import { Fragment, PureComponent } from 'react';
+import FixedNavigationHeader from 'calypso/components/fixed-navigation-header';
+
+export default class FixedNavigationHeaderExample extends PureComponent {
+	static displayName = 'FormattedHeaderExample';
+
+	render() {
+		return (
+			<Fragment>
+				<FixedNavigationHeader headerText="This is the header." />
+				<FixedNavigationHeader headerText="This is the header with branded font" brandFont />
+			</Fragment>
+		);
+	}
+}
```

Comment:
Nitpicking: What do you think about adding this example component as a functional one? Considering this is the path we want to reach in the future.

```js
export default function FixedNavigationHeaderExample () {
	const displayName = 'FormattedHeaderExample'; // do we use this?

	return (
		<Fragment>
			<FixedNavigationHeader headerText="This is the header." />
			<FixedNavigationHeader headerText="This is the header with branded font" brandFont />
		</Fragment>
	);
}

```

Reply:
oh 🤦 , that was a copy - paste remnant!  Fixed in e3849ba

Re `displayName`, no we shouldn't need it https://reactjs.org/docs/react-component.html#displayname